python/ray/tests/test_advanced.py

```diff
@@ -629,7 +627,6 @@ def test_task_arguments_inline_bytes_limit(ray_start_cluster):
             # max_grpc_message_size, this test fails.
             "task_rpc_inlined_bytes_limit": 18 * 1024,
             "max_grpc_message_size": 20 * 1024,
-            "put_small_object_in_memory_store": True,
```

Comment:
Ditto here, this is dealing with task argument inlining, not put inlining, so removing this should be fine.